packages/action-menu/README.md

```diff
@@ -6,7 +6,7 @@
     height="250px"
```

Comment:
Not as a comment on your work, but we should probably export the dummyModule so other people can get started aswell :).

Reply:
What do you mean exactly by this @blu3beri?

Reply:
We have a dummyModule in AFJ explaining to people on how to create an extension module, the old way. If we update that everyone has a clear example to start modularising other modules and their custom modules that they might use.

Reply:
Ah gotcha! Yes totally agree